nova/virt/vmwareapi/driver.py

```diff
@@ -358,6 +362,81 @@ def check_can_live_migrate_destination(self, context, instance,
         data = objects.VMwareLiveMigrateData()
         data.cluster_name = CONF.vmware.cluster_name
         data.datastore_regex = CONF.vmware.datastore_regex
+        data.host_ip = CONF.vmware.host_ip
+        data.host_username = CONF.vmware.host_username
+        data.host_password = CONF.vmware.host_password
+        data.instance_uuid = self._session.vim.service_content.about.instanceUuid
+        cert = ssl.get_server_certificate((CONF.vmware.host_ip, 443))
+        x509 = OpenSSL.crypto.load_certificate(OpenSSL.crypto.FILETYPE_PEM, cert)
```

Comment:
Could you check, if it is possible to get the fingerprint without the OpenSSL package?